### ISSUE TITLE
[MCHANGES-271] Web page outpdated since Maven 3

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-changes-plugin#556]
**[Markus Karg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mkarg)** opened **[MCHANGES-271](https://issues.apache.org/jira/browse/MCHANGES-271?redirect=false)** and commented

This page is outdated: http://maven.apache.org/plugins/maven-changes-plugin/usage.html

Since Maven 3 there is no \<reporting> section in pom.xml anymore. Please update page according to Maven 3 structure to no confure beginners with old configuration.

---

**Affects:** 2.6

**Comment by jira-importer** [comment]
**[Dennis Lundberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dennisl@apache.org)** commented

Configuring the Changes Plugin in the \<reporting> section still works with Maven 3. I'm not willing to change that part of the documentation until this plugin only work with Maven 3, which will probably be a long time from now.

**Comment by jira-importer** [comment]
**[Markus Karg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mkarg)** commented

This is not true. I just copied the configuration posted on the web site into a Maven 3.0.3's pom.xml and did mvn site. The result was I got my site, but no Javadoc report. It seems, Maven 3.0.3 just ignores \<reporting>. So I configured the pom.xml according to the migration guide and then it worked. That's why I posted this as "critical". If you resist from at least adding an additional example in the docs, people (like me) will spend lots of time for nothing. This I cannot understand your point of view, sorry.
